karyon3-core/src/main/java/com/netflix/karyon/http/ServerConfig.java

```diff
@@ -1,9 +1,9 @@
-package com.netflix.karyon.rxnetty;
+package com.netflix.karyon.http;
 
 import com.netflix.archaius.annotations.Configuration;
 import com.netflix.archaius.annotations.DefaultValue;
 
-@Configuration(prefix="karyon.rxnetty")
+@Configuration(prefix="karyon.httpserver")
```

Comment:
Does karyon have any notion of clients? Not a big deal, but having the configuration prefix correspond to the package can help make it easier to track back and find the usage of settings.